.gitignore

```diff
@@ -1,5 +1,7 @@
 node_modules/
 
+dist/
```

Comment:
📣 `npm publish` does not include files which are git ignored, hence you should not do that.
Feel free to double check by running `npm publish --dry` and check the list of files that it would publish.

Reply:
I believe you. It's annoying that the `dist` directory is always generated and untracked after building though :(